packages/core/src/mpi/get-patient-by-demo.ts

```diff
@@ -58,7 +59,17 @@ export const getPatientByDemo = async ({
   );
 
   if (matchingPatients.length > 1) {
-    log("WARNING: matchPatients in getPatientByDemo returning more than one matched patient!");
+    log(
+      `WARNING: matchPatients in getPatientByDemo for demo: ${demo} and cxId: ${cxId} is returning more than one matched patient!`
```

Comment:
Prob should have `demo` at the end w/ `JSON.stringify(demo)` - please avoid multi-line logs as CW breaks into multiple log entries.

Reply:
Done.